i18n/en.toml

```diff
@@ -1,5 +1,35 @@
 # i18n strings for the English (main) site.
 
+[deprecation_warning]
+other = " documentation is no longer actively maintained. The version you are currently viewing is a static snapshot. For up-to-date documentation, see "
+
+[objectives_heading]
+other = "Objectives"
+
+[cleanup_heading]
+other = "Cleaning up"
+
+[prerequisites_heading]
+other = "Before you begin"
+
+[whatsnext_heading]
+other = "What's next"
+
+[main_feedback]
+other = "Feedback"
+
+[main_feedback_question]
+other = "Was this page helpful?"
+
+[main_feedback_yes]
+other = "Yes"
+
```

Comment:
@tengqm, looks good. Is there a reason why the variables are prefixed with `main_`?